pkg/orchestrator/orchestrator.go

```diff
@@ -56,14 +55,17 @@ type OrchestratorSettings struct {
 	JenkinsUser  string
 	JenkinsToken string
 	AzureToken   string
+	GitHubToken  string
 }
 
 func NewOrchestratorSpecificConfigProvider() (OrchestratorSpecificConfigProviding, error) {
 	switch DetectOrchestrator() {
 	case AzureDevOps:
 		return &AzureDevOpsConfigProvider{}, nil
 	case GitHubActions:
-		return &GitHubActionsConfigProvider{}, nil
+		return initGHProvider(&OrchestratorSettings{
```

Comment:
Suggestion to stick to naming e.g. gitHubActionsConfigProvider{} instead of initGHProvider?